test/links.js

```diff
@@ -34,6 +37,63 @@ describe('Router#routeLinks', () => {
     assert.equal(called, 0, 'matching route should not be called')
   })
 
+  it('ignores links that have a target attribute', async () => {
+    let called = 0
+    let router = new Router({ routeLinks: true })
+      .use('/start', ({ resolve }) => resolve())
+      .use('/linked/:to', ({ params, resolve }) => {
+        ++called
+        assert.fail('should not route due to target attribute')
+      })
+
+    history.replaceState(null, document.title, '/start')
+    await router.start()
+
+    clickTo('#/linked/location', false, { target: '_self' })
+    await router.routing
+
+    router.stop()
```

Comment:
Can you assert the `location.hash` was updated, so we know that the default behavior was not prevented?